css/index.css

```diff
@@ -278,6 +278,13 @@ footer {
     font-size: 0.9rem;
   }
 
+  .card {
+    padding: 15px 1px;
```

Comment:
Can we make it 2px ?? 1px is a bit negligible

Reply:
For this it's not responsive